questionnaires/vaping_weekly_survey/vaping_weekly_survey_armt.json

```diff
@@ -0,0 +1,856 @@
+        "field_name": "cough",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 I never cough "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 I cough all the time"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "cough_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "phlegm",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 I have no phlegm (mucus) on my chest at all "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 My chest is full of phlegm (mucus)"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "phlegm_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "chest",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item below, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 My chest does not feel tight at all "
```

Comment:
I think labels might be a tad too long, might consider moving some text to field_label. Not sure they would be displayed very well with a range type

Reply:
Would it be ok to rephrase and make some changes (so they can test) or should we ask the clinicians first?

Reply:
fine by me since it is just for testing

Reply:
These were taken from the COPD Assessment Test, but rephrasing where needed is okay as we are not using the full measure anyway.